src/plugin/MapboxMapGestureArea.qml

```diff
@@ -118,7 +121,8 @@ Item {
 
             //! When released, indicate that touching has finished
             onReleased: {
-                __isTouching = false
+                __isTouching = false;
+				mpbxGestureArea.released(mouse);
```

Comment:
it looks to have tabs here instead of spaces. if you can correct it, I'll merge and build a new version at obs

Reply:
I did that with the online editor, should be OK now.